packages/wallets/wallet-turnkey/src/strategy/turnkey/oauth.ts

```
@@ -0,0 +1,93 @@
+      })
+    }
+  }
+
+  static async oauthLogin(args: {
+    client: HttpRestClient
+    oidcToken: string
+    providerName: 'google' | 'apple'
+    expirationSeconds?: number
+    iframeClient: TurnkeyIframeClient
+    oauthLoginPath?: string
+  }): Promise<{ credentialBundle: string } | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)